### ISSUE TITLE
Question in coding dmcnn.jpynb

### ISSUE THREAD

**Issue by caoxiaole** [EemeliSaari/dmcnn-vd#1]
Hi, I have a question in  coding dmcnn.  The size of 'outputs' after dmcnn model is [1024, 3, 21, 21]. While the size of 'target' is still [1024, 3, 32, 32]. The sizes of them are not matched. How to get the loss, which is loss = criterion(outputs , target) ?

**Comment by EemeliSaari** [comment]
Hey, sorry for late answer: The ImagePatchDataset class still has few parts where the input size is hard coded. I think changing some of the 32, 32 -> 21, 21 might resolve this.

**Comment by vonJJ** [comment]
Hello, I want to know that if you have pre-trained parameters for this dmcnn？ Thanks！

**Comment by EemeliSaari** [comment]
Sorry no pre-trained models available. As it seems for now the original authors never released their models ether.
